### ISSUE TITLE
Rename to `js-object-this`

### ISSUE THREAD

**Issue by gaand** [ga-wdi-boston/js-objects-this#17]
Not plural

**Comment by payne-chris-r** [comment]
I understand what you're saying, but I kind of disagree. I see this as another lesson in the "js-objects" saga (if you will). We also have js-objects, js-objects-prototypes, and others. So, I just see it as the next lesson describing types of js-objects. What are your thoughts @gaand and @jrhorn424 ?

**Comment by jrhorn424** [comment]
I agree that we should maintain consistency with the rest of the `js-objects` lessons. We can discuss a broader renaming if that makes sense, as well.

**Comment by gaand** [comment]
I prefer the plural for the other material, but the semantics of `this` are why I wanted to rename this rwpo. I may actually have a strong opinion.

**Comment by payne-chris-r** [comment]
Let's chat about this during today's planning @jrhorn424, @gaand

**Comment by gaand** [comment]
I've changed my mind and now would prefer `js-functions-this`

**Comment by jrhorn424** [comment]
I don't like that. Primary reason being that `js-functions-this` is the appropriate name of this lesson: https://github.com/ga-wdi-boston/js-context-this

The current repository is entirely descoped to only using `this` to refer to own properties.

**Comment by gaand** [comment]
Agreed. I opened https://github.com/ga-wdi-boston/js-context-this/issues/17

**Comment by gaand** [comment]
So, singular versus plural is still outstanding. 
I suggest an I person discussion to hash it out.

**Comment by BenGitsCode** [comment]
@ga-wdi-boston/core someone with permissions is required to make this change.
